pypeit/reduce.py

```diff
@@ -1097,22 +1116,28 @@ def local_skysub_extract(self, global_sky, sobjs, spat_pix=None, model_noise=Tru
                 # True  = Good, False = Bad for inmask
                 ingpm = (self.sciImg.fullmask == 0) & thismask
                 # Local sky subtraction and extraction
-                self.skymodel[thismask], self.objmodel[thismask], self.ivarmodel[thismask], \
-                    self.extractmask[thismask] = skysub.local_skysub_extract(
-                    self.sciImg.image, self.sciImg.ivar, self.tilts, self.waveimg,
-                    self.global_sky, self.sciImg.rn2img,
-                    thismask, self.slits_left[:,slit_idx], self.slits_right[:, slit_idx],
-                    self.sobjs[thisobj], ingpm,
-                    spat_pix=spat_pix,
-                    model_full_slit=self.par['reduce']['extraction']['model_full_slit'],
-                    box_rad=self.par['reduce']['extraction']['boxcar_radius']/self.get_platescale(None),
-                    sigrej=self.par['reduce']['skysub']['sky_sigrej'],
-                    model_noise=model_noise, std=self.std_redux,
-                    bsp=self.par['reduce']['skysub']['bspline_spacing'],
-                    sn_gauss=self.par['reduce']['extraction']['sn_gauss'],
-                    show_profile=show_profile,
-                    use_2dmodel_mask=self.par['reduce']['extraction']['use_2dmodel_mask'],
-                    no_local_sky=self.par['reduce']['skysub']['no_local_sky'])
+                try:
```

Comment:
Please remove this try/except, looks like something being used to debug.

Reply:
Done